docker/dev/Dockerfile-web

```diff
@@ -0,0 +1,32 @@
+FROM python:3.6.8-jessie
+
+RUN mkdir -p /app/panelapp
+
+COPY .build/panelapp /app/panelapp
+COPY .build/setup.py .build/setup.cfg .build/MANIFEST.in .build/VERSION /app/
+
+WORKDIR /app
+
+# Install everything in editable mode, for development
+RUN pip install -e .[dev,tests]
+
+## Prepare installation for development
+
+# Install dev-time dependencies
+RUN apt-get update && apt-get install build-essential -y
+
+# Copy the genes file
+ADD .build/genes.json.tgz /app
```

Comment:
this file is only required once, during the initial loaddata run

Reply:
This is a local-development only image. It must be easy to initialise the full environment from scratch. To be able to run `loaddata` we either mount it as a volume or include in the image. Neither is particularly lightweight but it is not a big issue, as this is not going to be shipped to any deployed environment.
It will definitely not be included in the production image